test/autoCompletion.test.ts

```diff
@@ -1591,6 +1591,20 @@ describe('Auto Completion Tests', () => {
       });
       assert.strictEqual(result.items.length, 3, `Expecting 3 items in completion but found ${result.items.length}`);
     });
+
+    it('should handle http(s) url', async () => {
+      const documentContent = `# yaml-language-server: $schema=http://json-schema.org/draft-07/schema# anothermodeline=value\n`;
```

Comment:
I assume this test is reaching out to the internet to grab the schema? If so, is it possible to mock the return of the schema so that we don't have to touch the internet